style.scss

```diff
@@ -0,0 +1,118 @@
+/* 
```

Comment:
Remove this file for this PR, then do a separate PR where you introduce a build step and port everything over to scss nah mean